src/jobs/helm_aws.yml

```diff
@@ -38,6 +38,10 @@ parameters:
     description: "Slack handles to mention on the failure notification"
     type: string
     default: ''
+  reset_dev_env_replica_count:
+    description: "Resets the replica count of each deployment in the specified dev env"
+    type: string
```

Comment:
this should be a boolean. Otherwise passing 'false' will also make it true which is unexpected